src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -188,6 +188,38 @@ public void Ctor_ExpectedDefaultPropertyValues_UapPlatform()
             }
         }
 
+        [Fact]
```

Comment:
This test should be outerloop since it using a remote Azure test endpoint.

Reply:
I added [OuterLoop] to it.